### ISSUE TITLE
Issue with DATSU on the 27th November 2023. How to trace back.

### ISSUE THREAD

**Issue by Osanna123** [ices-eg/DIG#615]

**Comment by Osanna123** [comment]
i could not find any info from my messages. If the issue is specific to the 27th of November only, DATSU was very popular with Intercatch, DATRAS, and had some sessions for SAG-CS, SAG, VMS test, and 1 BioBirds screening.
Sessions ran between 33726 to 33868.
What looks odd is that there were no screenings registered between 24/11/2023 15:46:00 and 27/11/2023 11:03:00, which seems quite odd compared to the activities on other days/times

**Comment by mikeydrew** [comment]
@odontaster after taking a look at the logs, here is what I found out about the Datsu/Datras issue. Most of the event logs are gone as the retention period(s) had expired. The event log for application errors was still available on DatrasLive01, but there were no related errors logged during this period. It was not possible to check the event logs on SQL10 as these had expired.
From the SQL logs on SQL10, there was an issue on the 25th of November in the morning for around 5 minutes, where the TempDB was full, which would have affected all the dbs on SQL10. The db was quickly shrunk, so this would not explain the issue lasting over 2 days. 
I checked the IIS logs on DatrasLive01, which log all the IIS activity to a specific website and could see the session you mentioned for 33700 on the 24th 
2023-11-24 11:20:32 172.17.44.174 GET /Data+submission/Screening_results.aspx SID=33700 80 - 165.225.17.168    
and then there were subsequent screenings up until 
2023-11-24 13:30:11 172.17.44.174 GET /Data+submission/Screening_results.aspx SID=30673 80
Then nothing else until 
2023-11-27 10:02:01 172.17.44.174 HEAD /Data+submission/Screening_results.aspx SID=33707 
As I can't see any other screening requests during this period, then my assumption is that there were no further requests - as I can see lots of other posts and gets to other Datras pages or the screening facility web functionality was down. If you are @vaishav are interested, I can show you the logs :)
